src/controllers/controller.cpp

```diff
@@ -121,7 +121,7 @@ void Controller::receive(const QByteArray data, mixxx::Duration timestamp) {
             continue;
         }
         function.append(".incomingData");
-        QScriptValue incomingData = m_pEngine->resolveFunction(function);
+        QScriptValue incomingData = m_pEngine->wrapFunctionCode(function, 2);
```

Comment:
This loop should be moved to: 
ControllerEngine::initializeScripts and fill a simple QList<QScripValue> m_incommongDataFunctions 
which is only called once. 

Here we can use than a simple .. without any string arithmetic: 
```
for (const auto& incomingData: m_pEngine->m_incommongDataFunctions()) {
        if (!m_pEngine->execute(incomingData, data, timestamp)) {
            qWarning() << "Controller: Invalid script function" << function;
        }
    }
```